### ISSUE TITLE
match_family cairo error recurring, reinstallation doesn't help.

### ISSUE THREAD

**Issue by zslastman** [davidgohel/gdtools#45]
I'm getting this error:

```> gdtools::match_family()
Error in match_family_(font, bold, italic) : 
  Fontconfig error: unable to match font pattern
Calls: <Anonymous> -> match_family_ -> .Call
Execution halted
```

I've tried reinstalling cairo with brew, and then reinstalling gdtools, doesn't help. Here is my session info:

```
R version 3.4.1 (2017-06-30)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS: /fast/users/dharnet_m/miniconda3/lib/R/lib/libRblas.so
LAPACK: /fast/users/dharnet_m/miniconda3/lib/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] data.table_1.11.4 magrittr_1.5      forcats_0.3.0     stringr_1.3.1    
 [5] dplyr_0.7.6       purrr_0.2.5       readr_1.1.1       tidyr_0.8.1      
 [9] tibble_1.4.2      ggplot2_3.0.0     tidyverse_1.2.1   colorout_1.1-2   

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.18     compiler_3.4.1   pillar_1.3.0     cellranger_1.1.0
 [5] plyr_1.8.4       bindr_0.1.1      tools_3.4.1      jsonlite_1.5    
 [9] lubridate_1.7.4  gtable_0.2.0     nlme_3.1-137     lattice_0.20-35 
[13] pkgconfig_2.0.1  rlang_0.2.1      cli_1.0.0        rstudioapi_0.7  
[17] haven_1.1.2      bindrcpp_0.2.2   withr_2.1.2      xml2_1.2.0      
[21] httr_1.3.1       hms_0.4.2        grid_3.4.1       tidyselect_0.2.4
[25] glue_1.3.0       R6_2.2.2         readxl_1.1.0     modelr_0.1.2    
[29] backports_1.1.2  scales_0.5.0     rvest_0.3.2      assertthat_0.2.0
[33] colorspace_1.3-2 stringi_1.2.3    lazyeval_0.2.1   munsell_0.5.0   
[37] broom_0.5.0      crayon_1.3.4    
```

**Comment by davidgohel** [comment]
Could you add the command you executed that generated the error? 

Also, could you add the result of `gdtools::sys_fonts()`?

**Comment by zslastman** [comment]
Hi David, thanks a million for the quick response. I managed to fix the error in the mean time, which was arising from:

```gdtools::match_family()```


I had somehow been cut off from access to the system fonts on the cluster I"m using, and  neither conda, nor devtools nor install.packages were installing them (which I'd assume it would, given they're a dependency). running fc-list in bash was giving me nothing. I fixed it by just copying tts files in manually from my laptop.

This is my problem rather than yours, but maybe someone else will find an error message that checks if fonts are actually available and suggests installing them if not to be useful.
